.github/workflows/check.yml

```diff
@@ -44,12 +44,14 @@ jobs:
         id: get-rust-version
         run: echo rust-version="$(rustc --version | awk '{print $2}' )" >> $GITHUB_OUTPUT
 
-      - name: Build alumet
-        run: "ls -l && cargo build --package alumet"
-        working-directory: alumet
+      - name: build app-agent
+        run: |
+          cd ./alumet/app-agent/ && cargo build
```

Comment:
Why not keeping working-directory: alumet/app-agent ?

Reply:
I need to compile the app agent in order to make the tests work. I will come back to a previous version

Reply:
Yes I know, but that's not my question...